src/js/tubular/models/tubular-column.js

```diff
@@ -10,16 +10,20 @@
          * @description
          * The `tubularColumn` factory is the base to generate a column model to use with `tbGrid`.
          */
-        .factory('tubularColumn', ['dataTypes', 'sortDirection', 
-        function (dataTypes, sortDirection) {
+        .factory('tubularColumn', ['dataTypes', 'sortDirection', 'aggregateFunctions', function (dataTypes, sortDirection, aggregateFunctions) {
             return function (columnName, options) {
                 options = options || {};
                 options.DataType = options.DataType || dataTypes.STRING;
+                options.Aggregate = options.Aggregate || aggregateFunctions.NONE;
 
                 if (Object.values(dataTypes).indexOf(options.DataType) < 0) {
                     throw `Invalid data type: '${options.DataType}' for column '${columnName}'`;
                 }
 
+                if (Object.values(aggregateFunctions).indexOf(options.Aggregate) < 0) {
```

Comment:
let's create a helper function IsIn for this...and use it on the line 19 as well